charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -42,9 +42,8 @@ spec:
         - --port
         - "8080"
         {{ if .Values.controllerManager.leaderElection.activated -}}
```

Comment:
It looks like this `if` never worked properly. `--leader-elect` is `false` by default so if `activated` is `true` then no command line option is passed and leader election is NOT enabled. Yes, the namespace option is set, but it does not matter. Nobody verified that part of the chart works as expected :)

Reply:
Are you sure about this? From my reading of the code, the default value of `--leader-elect` is true.

https://github.com/kubernetes-incubator/service-catalog/blob/770fc74f3f7120defc6b3a53a73a4b3e12425928/cmd/controller-manager/app/options/options.go#L79

Reply:
Yes, you are right! And I swear I saw this line! Consequences of coding at 11pm.